en/getting_started/quick_start_ubuntu_webcam.md

```diff
@@ -28,9 +28,8 @@ This Quickstart shows how to build CSD on Ubuntu LTS 16.04 (with support for MAV
    ```
    make
    ```
-1. Attach supported cameras to Ubuntu.
-   > **Note** Any camera that supports the [Video4Linux (V4L2) API](https://linuxtv.org/downloads/v4l-dvb-apis/uapi/v4l/v4l2.html) should be automatically detected. Some readily-available cameras that are known to work are: *Logitech C270 HD Webcam*, *Sony PlayStation Eye Camera*, *Intel RealSense*.
-1. Run CSD specifying Ubuntu [configuration file](../getting_started/building_installation.md#configuration-file-runtime).
+1. Attach [CSD-compatible cameras](../guide/overview.md#supported_cameras) to Ubuntu.
```

Comment:
FYI, moved camera list to overview - makes it easier to extend in future.

Reply:
The sequence is now correct. CSD will scan for the cameras on startup. For now, CSD will not add cameras plugged in at run time. For any new camera device to be detected, CSD will need to be restarted.